clarity-ext-scripts/clarity_ext_scripts/covid/fetch_biobank_barcodes.py

```diff
@@ -74,8 +79,9 @@ def biobank_barcode_by_sample_referral_code(self):
         file_stream = self.context.local_shared_file(RAW_BIOANK_LIST)
         biobank_matrix = self._build_biobank_info_by_well_barcode(file_stream)
         plate_barcode = self._plate_barcode_from(biobank_matrix)
-        file_stream2 = self.context.local_shared_file('Raw sample list')
-        sample_matrix = self._build_sample_info_by_well_barcode(file_stream2, plate_barcode)
+        file_stream2 = self.get_raw_sample_list(self.context)
+        sample_matrix = self._build_sample_info_by_well_barcode(
```

Comment:
file_stream2 is supposed to be a file stream. The output from get_raw_sample_list() is a panda object.